styles/stepper.css

```diff
@@ -9,7 +9,7 @@
     margin-bottom: 16px;
     margin-left: 10px;
 }
-  
+
```

Comment:
Still not sure if this is needed as part of the PR if there are no changes to this file